bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -0,0 +1,192 @@
+			<parameter name="refresh" type="integer" min="10" required="false" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the Refresh Interval in Seconds</description>
+				<default>10</default>
+				<unitLabel>s</unitLabel>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Roku TV Thing -->
+	<thing-type id="roku_tv">
+		<label>Roku TV</label>
+		<description>
+			A Roku Streaming Media TV
+		</description>
+
+		<channels>
+			<channel id="activeApp" typeId="activeApp"/>
+			<channel id="button" typeId="buttonTv"/>
+			<channel id="playMode" typeId="playMode"/>
+			<channel id="timeElapsed" typeId="timeElapsed"/>
+			<channel id="timeTotal" typeId="timeTotal"/>
+		</channels>
+
+		<properties>
+			<property name="Model Name">unknown</property>
+			<property name="Model Number">unknown</property>
+			<property name="Device Location">unknown</property>
+			<property name="Serial Number">unknown</property>
+			<property name="Device Id">unknown</property>
+			<property name="Software Version">unknown</property>
+		</properties>
+
+		<representation-property>uuid</representation-property>
+
+		<config-description>
+			<parameter name="hostName" type="text" required="true">
+				<context>network-address</context>
+				<label>Host Name/IP Address</label>
+				<description>Host Name or IP Address of the Roku TV</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535" required="true">
+				<label>Port</label>
+				<description>Port for the ECP Connector of the Roku TV</description>
+				<default>8060</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refresh" type="integer" min="10" required="false" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the Refresh Interval in Seconds</description>
+				<default>10</default>
+				<unitLabel>s</unitLabel>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="button">
+		<item-type>String</item-type>
+		<label>Remote Button</label>
+		<description>A Remote Button Press to Send to the Roku</description>
+		<state>
+			<options>
+				<option value="Home">Home</option>
+				<option value="Rev">Reverse</option>
+				<option value="Fwd">Forward</option>
+				<option value="Play">Play</option>
+				<option value="Select">Select</option>
+				<option value="Left">Left</option>
+				<option value="Right">Right</option>
+				<option value="Down">Down</option>
+				<option value="Up">Up</option>
+				<option value="Back">Back</option>
+				<option value="InstantReplay">Instant Replay</option>
+				<option value="Info">Info</option>
+				<option value="Backspace">Backspace</option>
+				<option value="Search">Search</option>
+				<option value="Enter">Enter</option>
+				<option value="FindRemote">Find Remote</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="buttonTv">
+		<item-type>String</item-type>
+		<label>Remote Button</label>
```

Comment:
You have multiple channels that have the same label. This causes an issue in OH3 as the label is what is used when you auto create the items using 'create equipement from thing' feature. Each label should be short and unique.

Reply:
>
> 
> You have multiple channels that have the same label. This causes an issue in OH3 as the label is what is used when you auto create the items using 'create equipment from thing' feature. Each label should be short and unique.

This was by design. One is used on standalone Roku things, the other is used on Roku TV things. The buttonTv channel adds additional button presets that only apply to TVs.